src-cpp/app/global/app_sections_config.h

```diff
@@ -0,0 +1,26 @@
+#ifndef APPSECTION_CONFIG_H
+#define APPSECTION_CONFIG_H
+
+#include <QString>
+
+const QString CHAT_SECTION_ID = "chat";
+const QString CHAT_SECTION_NAME = "Chat";
+const QString CHAT_SECTION_ICON = "chat";
+
+const QString WALLET_SECTION_ID = "wallet";
+const QString WALLET_SECTION_NAME = "Wallet";
+const QString WALLET_SECTION_ICON = "wallet";
+
+const QString BROWSER_SECTION_ID = "browser";
+const QString BROWSER_SECTION_NAME = "Browser";
+const QString BROWSER_SECTION_ICON = "browser";
+
+const QString NODEMANAGEMENT_SECTION_ID = "nodeManagement";
+const QString NODEMANAGEMENT_SECTION_NAME = "Node Management";
+const QString NODEMANAGEMENT_SECTION_ICON = "node";
+
+const QString SETTINGS_SECTION_ID = "profileSettings";
```

Comment:
I have simply taken this over from the nim code. dont want to change cause the UI is same for now, and may need these exact values when showing stuff, maybe once some implements Settings, it can be changed?

Reply:
do you know that every include should create personal instance of every each object here? it's a C++ you can use enum for it.

Reply:
Will add a to-do to improve this for however works on this later for now, I only want to bring up Wallet as of now.